server/server.js

```diff
@@ -1,15 +1,60 @@
 const express = require("express");
 const app = express();
+const cors = require("cors");
 const port = process.env.PORT || 5000;
+const videos = require("./exampleresponse.json"); 
```

Comment:
This file seems to be missing from your server directory, which causes the error `Error: Cannot find module './exampleresponse.json'` when starting the server

Reply:
Thank you for your help. It is working now